app/models/miq_policy.rb

```diff
@@ -95,6 +97,10 @@ def self.clean_attrs(attrs)
     attrs
   end
 
+  def display_name
+    @display_name ||= "#{towhat.constantize.display_name} #{mode.capitalize}: #{description}"
```

Comment:
@mzazrivec I assume that this would have the same effect as this:

https://github.com/ManageIQ/manageiq-ui-classic/blob/60abc4f0abfdb631d57bcfe7b1e299e2d6e337b0/app/controllers/miq_policy_set_controller.rb#L132

The intent base this off of what you were talking about here:

https://github.com/ManageIQ/manageiq/issues/20900#issuecomment-768967738

But let me know if this is not what you were suggesting there.

Reply:
I think we should remove the memoization.  For one, it's not expensive to rebuild on demand.  But also, if you change a value, it's suddenly wrong, which is unexpected...

Pseudo-code:

```ruby
x = MiqPolicy.create!
x.display_name  # Shows original description
x.description = "something else"
x.display_name  # Shows original description again
```

Reply:
@Fryguy Good call.  Fixed.